mappings/net/minecraft/fluid/Fluid.mapping

```diff
@@ -4,6 +4,7 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 	FIELD field_15905 stateManager Lnet/minecraft/class_2689;
 	METHOD method_15774 getBucketItem ()Lnet/minecraft/class_1792;
 	METHOD method_15775 appendProperties (Lnet/minecraft/class_2689$class_2690;)V
+		ARG 1 states
```

Comment:
```suggestion
		ARG 1 builder
```
The builder isn't really a list of states but a list of properties, and `Block.appendProperties` already uses `builder` as well.